client/src/store/reducers/products.js

```diff
@@ -0,0 +1,10 @@
+const LOAD_PRODUCTS = 'LOAD_PRODUCTS'
```

Comment:
This is repetitive, it can be saved in a constant file, and import to action creators and reducers when needed